Dalamud/Game/ClientState/ClientState.cs

```diff
@@ -64,22 +63,22 @@ private ClientState(SigScanner sigScanner, DalamudStartInfo startInfo, GameLifec
     private delegate IntPtr SetupTerritoryTypeDelegate(IntPtr manager, ushort terriType);
 
     /// <inheritdoc/>
-    public event EventHandler<ushort> TerritoryChanged;
+    public event Action<ushort>? TerritoryChanged;
 
     /// <inheritdoc/>
-    public event EventHandler Login;
+    public event Action? Login;
```

Comment:
Can you please clarify why you're switching this to an `Action`?

Reply:
EventHandler provides the function signature of `public void EventHandler(object sender, EventArgs args)`

Prior to this proposed PR that delegate was invoked using `this.Login?.InvokeSafely(this, null);` which is very redundant.
The consumer of this API already has the ability to use `ClientState` as you must have a pluginscoped instance to even subscribe to the event. Additionally passing null to args makes it a meaningless parameter.

This logic was applied to all the other provided events in this API aswell. These events will not provide the necessary information without including unused parameters.